moped-editor/src/views/projects/projectView/ProjectComponentsMap.js

```diff
@@ -328,39 +329,25 @@ const ProjectComponentsMap = ({
    * @param {Object} viewport - Mapbox object that stores properties of the map view
    */
   const handleViewportChange = useCallback(
-    viewport => setViewport(prevViewport => ({ ...prevViewport, ...viewport })),
+    (viewport) =>
+      setViewport((prevViewport) => ({ ...prevViewport, ...viewport })),
     [setViewport]
   );
 
-  /**
-   * Updates viewport on select of location from geocoder form
-   * @param {Object} newViewport - Mapbox object that stores updated location for viewport
-   */
-  const handleGeocoderViewportChange = useCallback(
-    newViewport => {
-      const geocoderDefaultOverrides = { transitionDuration: 1000 };
-
-      return handleViewportChange({
-        ...newViewport,
-        ...geocoderDefaultOverrides,
-      });
-    },
-    [handleViewportChange]
-  );
-
   /**
    * Customize cursor depending on user actions
    * @param {object} pointerStates - Object containing pointer state keys and boolean values
    * @param {boolean} pointerStates.isHovering - Is user hovering an interactive feature
    * @param {boolean} pointerStates.isDragging - Is user dragging map
    */
-  const getCursor = ({ isHovering, isDragging }) => {
-    return isDragging
-      ? "grabbing"
-      : isHovering || isDrawing // Show pointer when user is drawing as well
-      ? "pointer"
-      : "default";
-  };
+  // const getCursor = ({ isHovering, isDragging }) => {
+  //   return isDragging
+  //     ? "grabbing"
+  //     : isHovering // Show pointer when user is drawing as well
+  //     ? // : isHovering || isDrawing // Show pointer when user is drawing as well
+  //       "pointer"
+  //     : "default";
+  // };
```

Comment:
This will be removed in the next PR, but I kept it around so I could reference what the cursor does when drawing.